transit_odp/organisation/querysets.py

```diff
@@ -341,6 +341,19 @@ def add_live_data(self):
             published_at=F("live_revision__published_at"),
         )
 
+    def add_modified_status_for_error(self):
+        """Annotates the queryset with 'live' status if the status is 'error'"""
+        return self.annotate(
+            status=Case(
+                When(
+                    Q(live_revision__status="error"),
+                    then=Value("live", output_field=CharField()),
+                ),
+                default=F("live_revision__status"),
+                output_field=CharField(),
+            )
+        )
+
     def add_pretty_status(self):
```

Comment:
Line 353 will need to be reverted, right?

Reply:
I think, we need the output field for the default..